chsdi/templates/htmlpopup/aktuelle_erdbeben.mako

```diff
@@ -28,7 +29,7 @@
         r = requests.get(url)
         response = r.content.decode('utf-8')
         # Consider any status other than 2xx as error or mapserver 200 responses with MapServer Message in body
-        if not r.status_code // 100 == 2 or "MapServer Message" in r.content:
+        if not r.status_code == 200 or "MapServer Message" in response:
```

Comment:
The comment in line 31 says "consider any status other than 2xx ...". Now only 200 will be OK. 201, 202 and similar will lead to "invalid service response". If that is the intention, then the comment in line 31 probably should be adapted to "200" instead of "2xx"?